docs/data-collection/pre-session.md

```diff
@@ -49,6 +49,12 @@ Instructions of operations to be performed before the participant arrival, **bef
 
     ![nuprep-tube](https://shop.neurospec.com/media/catalog/product/cache/8af976a03d45f9b6d7f91e69feeeffb9/w/e/weaver_nuprep-skin-prep-gel-single-tube-front.jpg)
 
+- [ ] Open a new issue in [the github repository](https://github.com/TheAxonLab/hcph-sops/) to collect comments and annotations about the session [ON WHICH COMPUTER SHOULD THIS BE DONE?]
+    - [ ] Under the section `Issues`, click on <span class="consolebutton green">New issue</span>
+    - [ ] This should lead you to a page that looks like this 
+        ![](../assets/images/session_issue_template.png)
```

Comment:
I had to fix some conflicts and for some reason the image never uploaded (and I can't find anywhere) - @celprov could you please "manually" upload the image to this PR/branch ?